### PR TITLE
Add content changes to WLH

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This adds latest changes to WLH sections in ENG/WLS and NI schemas as described on this [Trello card](https://trello.com/c/AgiPdvOV). `Cancel and return` link change from the attached Google Sheet is done in [this runner PR](https://github.com/ONSdigital/eq-questionnaire-runner/pull/111).
### How to review 
Check the overflow to make sure the change is correctly implemented. Check Google sheet attached to the card to compare individual changes.

### CODE REVIEW

**Comment by petechd** [comment]
[[B] Add content changes to WLH (S)](https://trello.com/c/AgiPdvOV/3799-b-add-content-changes-to-wlh-s)

File: source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_temp_away_list_collector.jsonnet [hunk 1]
```diff
@@ -52,7 +52,7 @@ local editQuestion(questionTitle) = {
   },
   remove_answer: {
     id: 'anyone-else-temp-away-remove-confirmation',
-    value: 'Yes, I want to remove this person',
+    value: 'Yes, remove this person',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_temp_away_list_collector.jsonnet:55]: The CCS schemas aren't mentioned in the card or the spreadsheet of content changes. Should it be?

File: source/jsonnet/england-wales/household/blocks/who-lives-here/primary_person_list_collector.jsonnet [hunk 2]
```diff
@@ -48,13 +48,18 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
     },
     definitions: [
       {
-        title: 'What does “usually live” mean?',
+        title: 'What we mean by “usually live”?',
         contents: [
           {
-            description: 'It means the address at which you generally spend most time. For most people, this will be their permanent or family home.',
+            description: {
+              text: 'This is often your permanent or family home.<br><br>If you have more than one address, include yourself at the home address where you spend most of your time. If you split your time equally then use the home address where you are staying overnight on {census_date}.',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/household/blocks/who-lives-here/primary_person_list_collector.jsonnet:55]: `<br>` tags shouldn't be used. This should be done with two description elements.

File: source/jsonnet/northern-ireland/household/blocks/who-lives-here/primary_person_list_collector.jsonnet [hunk 3]
```diff
@@ -50,13 +50,18 @@ local questionTitle = {
     title: questionTitle,
     definitions: [
       {
-        title: 'What does “usually live” mean?',
+        title: 'What we mean by “usually live”?',
         contents: [
           {
-            description: 'It means the address at which you generally spend most time. For most people, this will be their permanent or family home.',
+            description: {
+              text: 'This is often your permanent or family home.<br><br>If you have more than one address, include yourself at the home address where you spend most of your time. If you split your time equally then use the home address where you are staying overnight on {census_date}.',
```
Comment (ajmaddaford) [inline source/jsonnet/northern-ireland/household/blocks/who-lives-here/primary_person_list_collector.jsonnet:57]: Same as previous comment about `<br>` tags.

Comment (petechd) [reply source/jsonnet/northern-ireland/household/blocks/who-lives-here/primary_person_list_collector.jsonnet]: Made it separate description, thanks 👍

File: source/jsonnet/england-wales/household/blocks/who-lives-here/primary_person_list_collector.jsonnet (hunk 2 above)
Comment (petechd) [reply source/jsonnet/england-wales/household/blocks/who-lives-here/primary_person_list_collector.jsonnet]: Made it separate description, thanks 👍

File: source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_temp_away_list_collector.jsonnet (hunk 1 above)
Comment (petechd) [reply source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_temp_away_list_collector.jsonnet]: Reverted the change in CCS, thanks 👍

**Pull request closed without merging**